@here/harp-mapview/test/TileGeometryCreatorTest.ts

```diff
@@ -355,68 +355,8 @@ describe("TileGeometryCreator", () => {
             expect(adapter1!.isPickable(new MapEnv({}))).to.equal(true);
         });
     });
-    it("categories", () => {
```

Comment:
I moved that test to the StyleSetEvaluatorTest, where the categories are processed now